plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -815,6 +819,21 @@ public Optional<ConnectorTableLayout> getNewTableLayout(ConnectorSession session
         return getWriteLayout(schema, partitionSpec, false);
     }
 
+    @Override
+    public Optional<io.trino.spi.type.Type> getSupportedType(ConnectorSession session, io.trino.spi.type.Type type)
+    {
+        if (type instanceof TimestampWithTimeZoneType) {
+            return Optional.of(TIMESTAMP_TZ_MICROS);
```

Comment:
> Support time types with precison <= 6 in Iceberg CT and CTAS

Change title 

- replace `time types` with `date/time types`
- replace `with precison <= 6` with `with precison != 6`

Reply:
i changed the PR title to much commit title.